app/test/unit/git/stash-test.ts

```diff
@@ -0,0 +1,29 @@
+import * as FSE from 'fs-extra'
+import * as path from 'path'
+import { Repository } from '../../../src/models/repository'
+import { setupEmptyRepository } from '../../helpers/repositories'
+import { GitProcess } from 'dugite'
+import { MagicStashString, getStashEntries } from '../../../src/lib/git/stash'
+
+describe('git/stash', () => {
+  describe('getStashEntries', () => {
+    it('returns all stash entries created by desktop', async () => {
+      const repository = await setupEmptyRepository()
+
+      await FSE.writeFile(path.join(repository.path, 'README.md'), 'Hi world\n')
+      await stash(repository)
+
+      const stashEntries = await getStashEntries(repository)
```

Comment:
This test is failing here...

![Alt Text](https://media.giphy.com/media/ANbD1CCdA3iI8/giphy.gif)

Not sure why 🤔 

**Error message**

Specifically the `GitError` bit.

```sh
~/Repositories/desktop/desktop(all-the-stashes*) » cd /Users/iamwill/Repositories/desktop/desktop ; env ELECTRON_RUN_AS_NODE=1 /Users/iamwill/Repositories/desktop/desktop/node_modules/.bin/electron --inspect-brk=35825 node_modules/jest/bin/jest.js --silent --config /Users/iamwill/Repositories/desktop/desktop/app/jest.unit.config.js app/test/unit/git/stash-test.ts 
Debugger listening on ws://127.0.0.1:35825/5cab2f8e-bc43-4966-9109-c77485c7401f
For help, see: https://nodejs.org/en/docs/inspector
Debugger attached.
 FAIL  app/test/unit/git/stash-test.ts
  ● git/stash › getStashEntries › returns all stash entries created by desktop

    GitError: fatal: ambiguous argument 'refs/stash': unknown revision or path not in the working tree.
    Use '--' to separate paths from revisions, like this:
    'git <command> [<revision>...] -- [<file>...]'

      171 |   log.error(errorMessage.join('\n'))
      172 | 
    > 173 |   throw new GitError(gitResult, args)
          |         ^
      174 | }
      175 | 
      176 | function getDescriptionForError(error: DugiteError): string {

      at Object.git (src/lib/git/core.ts:173:9)

Test Suites: 1 failed, 1 total
Tests:       1 failed, 1 total
Snapshots:   0 total
Time:        3.326s
```